aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/forms/controls/MultiSelectInput.get.html.ftl

```diff
@@ -0,0 +1 @@
+<@processJsonModel group="share"/>
```

Comment:
Keep meaning to say, but we don't need the group="share" anymore, that's a hangover from when the code was part of Share - you don't necessarily need to remove it (there are lots of other examples where this is still present) but be aware that it's no longer necessary)